doc/keymap-format-text-v1.md

```diff
@@ -351,11 +351,98 @@ satisfied the LED is lit.
 
 ## The `xkb_symbols` section
 
-This section is the fourth to be processed, after `xkb_keycodes`,
-`xkb_types` and `xkb_compat`.
+This section is the fourth to be processed, after `xkb_keycodes`, `xkb_types`
+and `xkb_compat`.
 
-TODO
+Statements of the form:
+
+    xkb_symbols "basic" {
+        ...
+    }
+
+Declares a symbols map named `basic`. Statements inside the curly braces will
+affect only affect the symbol map.
+
+Options can be specified above the statement, separated by one space:
```

Comment:
The code calls these "flags"; I prefer we use this term instead of "options" because there is already another concept called "options" and it's probably better to use distinct terms when possible to reduce the possibility of confusion.

Also I don't think it needs to be separated by exactly one space really.


```suggestion
A map can have various flags applied to it above the statement, separated by whitespace:
```